src/features/movie/components/imageContainer/ImageContainer.tsx

```diff
@@ -0,0 +1,43 @@
+import { MovieProps } from "../../../../models/movie.model";
+
+import { AiOutlineStar } from "react-icons/ai";
+import imgIron from "../../../../assets/images/iron.jpg";
+
+import styled from "./imageContainer.module.scss";
+
+export const ImageContainer: React.FC<MovieProps> = (movie) => {
```

Comment:
Naming... naming... naming... :) 

Naming here could be better. `ImageContainer` implies a generic component, **but** it's receiving a movie as a parameter and has declared MovieProps - meaning its coupled to movie, and is some kind of movie image container. Please rename it to communicate this better.

Reply:
MovieHeader?